### PR TITLE
Update nodejs config for cognitive services

### PR DESCRIPTION
**Pull request by RikkiGibson**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-entitysearch
-  package-version: 1.0.0-preview
+  package-version: 1.1.0
```
Comment (RikkiGibson) [inline specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md]: Note that the version we've actually published is 1.1.0--not likely to publish a new one as it looks like swagger has not changed. Just doing a correction here

File: specification/cognitiveservices/data-plane/ImageSearch/readme.nodejs.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-imagesearch
-  package-version: 1.0.0-preview
+  package-version: 1.0.0
```
Comment (hovsepm) [inline specification/cognitiveservices/data-plane/ImageSearch/readme.nodejs.md]: Just curious, are these parts of Cognitive service live separately? So there is a package for VideoSearch and then another one for VisualSearch?

Comment (RikkiGibson) [reply specification/cognitiveservices/data-plane/ImageSearch/readme.nodejs.md]: Yeah, we publish a different npm package for each one.

**Review by hovsepm — APPROVED** [review]

**Comment by hovsepm** [comment]
Waiting for @daschult to take a look...

**Comment by RikkiGibson** [comment]
@daschult not much to see here but do you mind giving a review?

**Pull request merged**